transformation/src/api/pipelineConfigTriggerRequest.test.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-env jest */
+import { PipelineConfigTriggerRequestValidator } from './pipelineConfigTriggerRequest'
+
+describe('PipelineConfigTriggerRequestValidator', () => {
+  test('should reject invalid trigger request', () => {
```

Comment:
I would divide this test into smaller test, so that it is clear what the problem is when a test fails.
In line 10-12 it is not obviously what is tested. 

```js
const validator = new PipelineConfigTriggerRequestValidator() 

test('should reject undefined request', () => {
    expect(validator.validate(undefined)).toBeFalsy()
})

test('should reject empty object or array', () => {
    expect(validator.validate([])).toBeFalsy()
    expect(validator.validate({})).toBeFalsy()
})

test('should reject ...', () => {
})


```